doc-source/notifiers.rst

```diff
@@ -145,6 +145,41 @@ Channel ID ``11223344`` can be obtained quickly from the URL to a
 message of the target private channel. Of course, the Slack App needs
 to be part of the private channel.
 
+PagerDuty
+---------
+
+This configurable notifier will trigger the PagerDuty.
+The following is an example configuration for this notifier
+to be added to a configuration file and used with the ``--pagerduty``
+option when executing your compliance check.
+Note that you have two options to configure the PagerDuty notifier:
+
+* Use a dictionary with different tests in your checker.
+In this way, you can modify your checker that only trigger the PagerDuty
+for a single test::
+
+  {
+    "pagerduty": {
+        "cloudant.infra.critical": {
+          "escalation": "XXXX",
+          "checks": [
+            "accred.Checker_Name1.test1",
+            "accred.Checker_Name2.test2
+          ]
+        }
+    }
+  }
+
+
+* Use a string, in this way the PagerDuty will trigger for all the tests
+belong to this accreditation::
+
+  {
+    "pagerduty": {
+      "accred": "ESCALATION"
+    }
+  }
```

Comment:
I think this needs a rewrite. dictionary v. string isn't clear as they're both JSON documents and both examples resolve to dictionaries.  We should also not have Cloudant specific examples.  Suggest something like:

```
* Provide a list of checks by class path within an accreditation.  This allows you 
to define which checks within the accreditation will trigger PageDuty notifications::

  {
    "pagerduty": {
      "my.accred1": {
        "escalation": "XXXX",
        "checks": [
          "package.category.checks.test_module_one.CheckClassOne",
          "package.category.checks.test_module_two.CheckClassTwo"
        ]
      }
    }
  }


* Provide accreditations only and the notifier will send alerts for all checks with those
accreditations.::

  {
    "pagerduty": {
      "accred": "ESCALATION"
    }
  }
```

Reply:
It's also unclear to me whether "accred" is supposed to represent a dynamic accreditation name or if it is always expected to be "accred".  If it's supposed to be dynamic then I suggest changing "accred" to "my.accred1".  Also if it's supposed to be dynamic then what does "ESCALATION" represent?

Reply:
Thanks AI, yes I think my.accred1 is good because it's supposed to be dynamic.